velox/dwio/parquet/tests/reader/ParquetTableScanTest.cpp

```diff
@@ -37,13 +37,12 @@ class ParquetTableScanTest : public HiveConnectorTestBase {
 
   void SetUp() {
     registerParquetReaderFactory();
+    HiveConnectorTestBase::SetUp();
```

Comment:
The `registerParquetReaderFactory` also happens in the updated `HiveConnectorTestBase::SetUp` so we don't need it here, right?

Reply:
fixed.